src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandler.java

```diff
@@ -499,8 +501,20 @@ private CallbackResponse gaValidateMakeAnOrder(CallbackParams callbackParams) {
         caseDataBuilder
             .judicialDecisionRequestMoreInfo(buildRequestMoreInfo(caseData, judgeNameTitle).build());
 
+        ArrayList<String> errors = new ArrayList<>();
+
+        if ((caseData.getApplicationIsUncloackedOnce() == null
```

Comment:
Try to check if possible to use GeneralAppInformOtherParty().getIsWithNotice() this variable to avoid addition of new variable.